packages/victory-area/src/helper-methods.js

```diff
@@ -58,6 +58,7 @@ const getBaseProps = (props, fallbackProps) => {
   const modifiedProps = Helpers.modifyProps(props, fallbackProps, "area");
   props = assign({}, modifiedProps, getCalculatedValues(modifiedProps));
   const {
+    ariaLabel,
```

Comment:
There should not be any changes in this file, since `ariaLabel` will only apply to primitives